lib/oli_web/live/admin/admin_view.ex

```diff
@@ -0,0 +1,79 @@
+defmodule OliWeb.Admin.AdminView do
+  use Surface.LiveView, layout: {OliWeb.LayoutView, "live.html"}
+  alias Oli.Repo
+  alias OliWeb.Common.Properties.{Groups, Group}
+  alias OliWeb.Common.Breadcrumb
+  alias Oli.Accounts.{Author}
+  alias OliWeb.Router.Helpers, as: Routes
+
+  prop author, :any
+  data breadcrumbs, :any
+  data title, :string, default: "Admin"
+
+  def mount(_, %{"current_author_id" => author_id}, socket) do
+    author = Repo.get(Author, author_id)
+
+    {:ok,
+     assign(socket,
+       author: author,
+       breadcrumbs: breadcrumb()
+     )}
+  end
+
+  def render(assigns) do
+    ~F"""
+    <Groups>
```

Comment:
seeing this makes me feel like it belongs in a template file, I presume there is a reason that its not

Reply:
Yes, indeed.  Although right this "Admin View" landing page contains no "dynamic" content and could be instead represented with a statically rendered template, I expect that there will be future functionality on this page.  So instead of having to reformulate it again in the future, I simply went with a LiveView implementation from the beginning.